backend/src/main/java/com/google/blackswan/mock/SimpleRelatedDataGenerator.java

```diff
@@ -81,17 +75,17 @@ public static SimpleRelatedDataGenerator createGenerator() {
   private void prefillRelatedData() {
     // TODO: Replace with call for querying configs from datastore.
     // TODO: Deal with capitalizations when querying config from datastore.
-    relatedDataMap.put(DataInfo.of(INTEREST_METRIC, "Ramen"), 
-        DataInfoUser.of(INTEREST_METRIC, "Udon", CONFIG_USERNAME));
-    relatedDataMap.put(DataInfo.of(INTEREST_METRIC, "Ramen"), 
-        DataInfoUser.of(INTEREST_METRIC, "Pho", CONFIG_USERNAME));
+    relatedDataMap.put(DataInfo.of(Const.INTEREST_US, Const.RAMEN), 
+        DataInfoUser.of(Const.INTEREST_US, Const.UDON, CONFIG_USERNAME));
+    relatedDataMap.put(DataInfo.of(Const.INTEREST_US, Const.RAMEN), 
+        DataInfoUser.of(Const.INTEREST_US, Const.PHO, CONFIG_USERNAME));
     // Multimap looks like this right now: 
     // {{Interest Level, Ramen},{{Interest Level, Udon}, {Interest Level, Pho}}}, ...}.
   }
 
   private ImmutableMap<Timestamp, Integer> getTopicDataPoints(DataInfo topic) {
     return ImmutableMap.copyOf(csvDataCache.computeIfAbsent(topic, key -> CSVParser.parseCSV(
-        SimpleRelatedDataGenerator.class.getResourceAsStream(FILE_LOCATIONS.get(key))
+        LocalFileSystem.createSystem().getDataAsStream(topic)
```

Comment:
do you mind explaining this change (from `SimpleRelatedDataGenerator` to `LocalFileSystem`

Reply:
`SimpleRelatedDataGenerator.class.getResourceAsStream(FILE_LOCATIONS.get(key))`
^what this line basically does is it finds the specified file name `FILE_LOCATIONS.get(key)` in `/resources` and reads the file in to an `InputStream` that is passed into `parseCSV(..)`.

Since I now want to use `LocalFileSystem`/`CloudFileSystem` to read the data and want the option of not reading it locally from the `/resources` folder, I replace the line above with the line that uses the `LocalFileSystem` which has the method `getDataAsStream` and takes in a `DataInfo topic` and returns an `InputStream`. `LocalFileSystem` has access to mapping between `DataInfo topic` and the filename/location, which is why here we pass in the `topic` instead of the filename and location.